.github/workflows/tests.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+    branches:
+      - develop
+
+name: Run tests
+
+jobs:
+
+  precommit:
+    name: Run the precommit checks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install deps
+        run: pip install click conu pytest pytest-cov munch psycopg2 fedora-messaging
+
+      # we only really need to run the precommit checks for one release,
+      # so we just do it for pip
+      - name: Run the precommit checks
+        run: devel/ci/bodhi-ci pre-commit -r pip
+  
+  
+  diff_cover:
+    name: Run the Diff Cover checks
```

Comment:
Maybe call it "Diff-cover check", to be consistent with the rest?

Reply:
updated. also updated the mergify commit to reflect this change